python/paddle/distribution/continuous_bernoulli.py

```diff
@@ -0,0 +1,446 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections.abc import Sequence
+
+import paddle
+from paddle.distribution import distribution
+
+
+class ContinuousBernoulli(distribution.Distribution):
+    r"""The Continuous Bernoulli distribution with parameter: `probability` characterizing the shape of the density function.
+    The Continuous Bernoulli distribution is defined on [0, 1], and it can be viewed as a continuous version of the Bernoulli distribution.
+
+    [1] Loaiza-Ganem, G., & Cunningham, J. P. The continuous Bernoulli: fixing a pervasive error in variational autoencoders. 2019.
+
+    Mathematical details
+
+    The probability density function (pdf) is
+
+    .. math::
+
+        p(x;\lambda) = C(\lambda)\lambda^x (1-\lambda)^{1-x}
+
+    In the above equation:
+
+    * :math:`x`: is continuous between 0 and 1
+    * :math:`probability = \lambda`: is the probability.
+    * :math:`C(\lambda)`: is the normalizing constant factor
+
+    .. math::
+
+        C(\lambda) =
+        \left\{
+        \begin{aligned}
+        &2 & \text{ if $\lambda = \frac{1}{2}$} \\
+        &\frac{2\tanh^{-1}(1-2\lambda)}{1 - 2\lambda} & \text{ otherwise}
+        \end{aligned}
+        \right.
+
+    Args:
+        probability(int|float|Tensor): The probability of Continuous Bernoulli distribution between [0, 1],
+        which characterize the shape of the pdf. If the input data type is int or float, the data type of
+        `probability` will be convert to a 1-D Tensor the paddle global default dtype.
+        eps(float): Specify the bandwith of the unstable calculation region near 0.5. The unstable calculation region
+        would be [0.5 - eps, 0.5 + eps], where the calculation is approximated by talyor expansion. The
+        default value is 0.02.
```

Comment:
```suggestion
    Args:
        probability(int|float|Tensor): The probability of Continuous Bernoulli distribution between [0, 1],
            which characterize the shape of the pdf. If the input data type is int or float, the data type of
            `probability` will be convert to a 1-D Tensor the paddle global default dtype.
        eps(float): Specify the bandwith of the unstable calculation region near 0.5. The unstable calculation region
            would be [0.5 - eps, 0.5 + eps], where the calculation is approximated by talyor expansion. The
            default value is 0.02.
```
对于 Args 下的每个参数，同一个参数的描述换行需要加下缩进

Reply:
已修改